woocommerce/woocommerce-android
# Support decimal quantities in Orders, and fix shipping labels handling of them

Closes #3808 and #4118.

Please don't merge before the FluxC's PR is merged: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2037

#### Testing
Please install the plugin [WPC Product Quantity for WooCommerce](https://wordpress.org/plugins/wpc-product-quantity/) in your store to use decimal quantities.

1. Create an order with decimal quantity > 1 in your store.
2. Go to order details in the app.
3. **Confirm** that you can see the correct quantity.
4. Click on "Create shipping label"
5. Pass the steps until you reach the packaging step.
6. **Confirm** that the items in the package are duplicated depending on their quantity.
7. Purchase the shipping label.
8. Open order details in the web client, and click on "View Details" of the label (https://d.pr/i/AmY0CH).
9. **Confirm** that the items match the correct quantity.
10. Go back to the order details in the app.
11. Click on "Issue a refund"
12. **Confirm** that the max quantity you can select is matching the integer part of the quantity.
13. Issue a refund with full max quantity.
14. In the order details: **Confirm** that the remaining quantity is correct, and that "issue a refund" is not visible

Update release notes:
